frontendActions/src/main/resources/actions/ResourceBundle.properties

```diff
@@ -5,18 +5,22 @@ action.GitMachete.CheckoutSelectedBranchAction.text=Checkout Selected Branch
 action.GitMachete.CheckoutSelectedBranchAction.GitMacheteContextMenu.text=_Checkout Branch
 action.GitMachete.CheckoutSelectedBranchAction.description=Checkout this branch
 
-action.GitMachete.RebaseSelectedBranchOntoParentAction.text=Rebase Selected Branch Onto Parent
-action.GitMachete.RebaseSelectedBranchOntoParentAction.GitMacheteContextMenu.text=_Rebase Branch Onto Parent
-action.GitMachete.RebaseSelectedBranchOntoParentAction.description=Rebase this branch onto parent
+action.GitMachete.RebaseSelectedBranchOntoParentAction.text=Checkout and Rebase Selected Branch Onto Parent
+action.GitMachete.RebaseSelectedBranchOntoParentAction.GitMacheteContextMenu.text=Checkout and _Rebase Branch Onto Parent
```

Comment:
As below, `Checkout and` redundant in case of current branch (can be resolved within a separate issue)

Reply:
Issue #327 created